app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+   def index
+    food_input = 'sweet potatoes'
+    @food_response = FoodFacade.foods(food_input)
```

Comment:
Careful with hardcoding your search. This search string should come from the search form on your welcome index page upon submission.